project_resource_calendar/wizard/weekly_report_wizard.py

```diff
@@ -0,0 +1,81 @@
+from datetime import datetime, timedelta
+
+from odoo import models, fields, api
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT as DATE_FORMAT
+
+
+class WeeklypReportWizard(models.TransientModel):
+    _name = 'weekly.report.wizard'
+
+    date_start = fields.Date(string="Start Date", required=True, default=fields.Date.today)
+    date_end = fields.Date(string="End Date", required=True, default=fields.Date.today)
+    room_id = fields.Many2one(
+        string='Room',
+        comodel_name='resource.calendar.room',
+        ondelete='set null',
+        required=True,
+    )
+    recurrency = fields.Boolean(
+        string='Recurrency',
+        default=True,
+        required=True,
+    )
+    state = fields.Selection([
+        ('draft', 'Unconfirmed'),
+        ('open', 'Confirmed'),
+        ('cancelled', 'Cancelled')],
+        string='Status',
+        default='open',
+        required=True,
```

Comment:
@eilst is it required? what if we want t display all the reservations?

Reply:
No, this is a spec